lib/find-view.coffee

```diff
@@ -447,21 +447,30 @@ class FindView extends View
     else
       optionButton.removeClass 'selected'
 
+  anySelectionsMatchFindPattern: =>
+    editor = @model.getEditor()
+    selections = editor?.getSelections() or []
+    _.any selections, (selection) =>
+      selectionMatchesFindPattern = false
+      editor.scanInBufferRange @model.getFindPatternRegex(), selection.getBufferRange(), ({range}) ->
```

Comment:
This scan seems unnecessary. The model should know if there is a current result marker IIRC. Seems like we should just be able make the decision based on the existence of the current result marker.